Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+Find.swift

```diff
@@ -223,14 +242,27 @@ extension PathHierarchy {
                 let possibleMatches = collisions.compactMap {
                     return try? $0.node.children[nextPathComponent.name]?.find(nextPathComponent.kind, nextPathComponent.hash)
                 }
+                let onlyPossibleMatch: Node?
```

Comment:
Would it be helpful to have a comment explaining what we're using this for? Or will that just clutter things?

Reply:
There's a comment broken up across multiple lined starting with 
```
// Try resolving the rest of the path for each collision ...
```
but now that the code has gotten longer I'll put it all back into one paragraph instead